src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EgressOperationStatus.cs

```diff
@@ -20,6 +20,24 @@ public class OperationSummary
 
         [JsonPropertyName("status")]
         public OperationState Status { get; set; }
+
+        [JsonPropertyName("processInfo")]
+        public OperationProcessInfo ProcessInfo { get; set; }
```

Comment:
Do we want to call this `ProcessInfo` in the returned result? Would it be simpler to just call it `Process`. I don't think the `Info` part conveys any more meaning and it's not something we've used in the rest of the HTTP API.

I'm also thinking longer term if we somehow collect an artifact from someone other mechanism and then we can add something like `"ArtifactSource": "Process"` as a property to indicate from what the artifact was collected.

Reply:
I'm a fan of changing it to `Process`. Done.